README.md

```diff
@@ -3,3 +3,63 @@
 This is the documentation for the upcoming release of the WSO2 Enterprise Integrator 7 family. Please note that this documentation is work in progress.
 
 To see the **latest released documentation** for the WSO2 Enterprise Integrator 7 family, go to: https://ei.docs.wso2.com/en/latest/
+
+## Contributing to WSO2 EI documentation
+
+As an open source project, WSO2 EI welcomes contributions from the community. To start contributing, read these contribution guidelines for information on how you should go about contributing to our project.
+
+1. Accept the Contributor License Agreement (CLA)
+    
+    You need to Accept the Contributor License Agreement (CLA) when prompted by a GitHub email notification after sending your first Pull Request (PR). Subsequent PRs will not require CLA acceptance.
+
+    If the CLA gets changed for some (unlikely) reason, you will be presented with the new CLA text after sending your first PR after the change.
+
+2. Fork this repo, make your changes, and send in a pull request. Make sure you are contributing to the correct branch (for example, if your change is relevant to WSO2 EI 7.0.0 documentation, you should commit your changes to the 7.0.0 branch).
+
+Check the issue tracker for open issues that interest you. We look forward to receiving your contributions.
+
+## Building the documentation locally
+
+### 1. Install Python. 
+
+If you are using MacOS, you probably already have a version of Python installed on your machine. You can verify this by running the following command.
+
+`$ python --version`
```

Comment:
Shall we remove the $ mark?
users can't copy the syntax because of this symbol

Reply:
Done.